1-js/11-async/04-promise-error-handling/article.md

```diff
@@ -1,21 +1,20 @@
+# Penanganan error dengan promise
 
-# Error handling with promises
+Chain promise bagus dalam penanganan _error_. ketika sebuah promise me-reject, kontrolnya melompat ke handler rejection terdekat. Itu sangat nyaman dalam praktiknya.
 
-Promise chains are great at error handling. When a promise rejects, the control jumps to the closest rejection handler. That's very convenient in practice.
-
-For instance, in the code below the URL to `fetch` is wrong (no such site) and `.catch` handles the error:
+Sebagai contoh, kode di bawah sebuah URL ke `fetch` itu salah (tidak ada situs seperti itu) dan `.catch` menangani error tersebut:
 
 ```js run
 *!*
-fetch('https://no-such-server.blabla') // rejects
+fetch('https://no-such-server.blabla') // reject
 */!*
   .then(response => response.json())
-  .catch(err => alert(err)) // TypeError: failed to fetch (the text may vary)
+  .catch(err => alert(err)) // TypeError: gagal untuk mengambil (teksnya mungkin berbeda)
```

Comment:
Kalo error kayanya jangan diterjemahin deh, tapi mungkin dikasih tambahan, kaya _TypeError: failed to fetch (gagal mengambil resourse, error yang dihasilkan mungkin berbeda)_